allegrord.cpp

```diff
@@ -631,7 +631,7 @@ double Alg_reader::parse_loud(string &field)
 {
     const char *msg = "Loudness expected";
     if (isdigit(field[1])) {
-        return parse_int(field);
+        return (double)(parse_int(field));
```

Comment:
I wrote earlier that after seeing there's really no double-to-int construct function in C++ that I can accept that the (ugly) static_cast<int> is arguably most correct. But in fact, both are safe even if the compiler (so far) is too limited to know that. I'm happy either way for this and the casts to int above.

Reply:
Keeping C-style cast here as well, resolving